flow/mappings/docker.go

```diff
@@ -0,0 +1,155 @@
+/*
```

Comment:
I think we should move this file to topology probe as it basically retrieves topology informations. We could add metadata information about docker on the veth node ? We already have the graph mapping in order to associate flow and intf attributes.